src/components/OrgUpcomingEvents.tsx

```diff
@@ -0,0 +1,38 @@
+import { api } from "@src/utils/api";
+import { type FC } from "react";
+import { useState} from "react";
+
+const OrgUpcomingEvents:FC<{club_id : string}> = ({club_id}) => {
+
+    const [cur_time] = useState( new Date() );
+
+    const query_events = (api.event.byClubId.useQuery({ clubId: club_id, currentTime: cur_time, sortByDate: true  })).data
```

Comment:
It would be a lot better if you could restructure the object rather than wrapping it in parens like that and then accessing data

Reply:
I have move those parameter to an object